api/question/views.py

```diff
@@ -37,14 +37,32 @@ def post(self):
 
     def get(self):
```

Comment:
@Tony-Ndichu  the problem is here, thanks for noting that, i think its fixed on the other prs